### ISSUE TITLE
Guidance on logo, name, etc usage

### ISSUE THREAD

**Issue by betatim** [jupyterhub/team-compass#267]
https://discourse.jupyter.org/t/treebeard-project-using-repo2docker/3600

We should come up with some guidelines for these use-cases. I think all our code is already covered (BSD3 licensed code). Not sure about our documentation (should be CC-by?? but is BSD3 license right now). The logo and name also have no guidance on how to use them.

**Comment by choldgraf** [comment]
Agreed - here's the jupyter trademarks page:

https://github.com/jupyter/governance/blob/master/trademarks.md

Perhaps we can either adapt this for Binder trademarks, or just point to it?

**Comment by betatim** [comment]
Maybe we can copy some of the guidelines. What makes this tricky is that repo2docker 9the word), its logo, binder (logo, basic word and word variations) are not trademarks we have registered.

Maybe NF gives advice on if/how to registered them.

For the time being we could take the guidelines and reformulate them so that they give advice on how we'd like things to be used without creating the impression that we have registered them as trademarks.

**Comment by choldgraf** [comment]
I'm +1 on CC-BY for documentation and non-code artifacts

**Comment by betatim** [comment]
Do you know how we'd transition from BSD to CC-BY for the docs? Do we list both in the `LICENSE` file of the repo with a comment which applies where? I don't have any experience with this :(

**Comment by choldgraf** [comment]
Yeah I'd probably just update the license file to say that the documentation is CC-BY, probably a one-liner would work fine.

If we moved to using a project-wide LICENSE, perhaps we could use this to udpate *every* project's license at once?

**Comment by betatim** [comment]
I'm wondering if we need to ask the original authors (as they hold the copyright) in order to relicense the content. This seems not mega feasible :-/

**Comment by choldgraf** [comment]
wouldn't the original content technically be BSD-3? How would this be different from just "fork-and-modify" the content?

**Comment by betatim** [comment]
If you didn't re-license it then yes all the old content would be BSD3 and the new content would be CC-BY. This weird mixture would be super confusing IMHO. Having this mixed license situation seems worse than the current (slightly weird license for the docs) because you now need to understand BSD3-for-content and CC-BY. To me that is more complicated than the current situation.

If we can work out a way to switch the license to CC-BY that would be worth doing because it simplifies things. However I am not sure what the procedure is.

Maybe @willingc has an idea or pointer?

**Comment by choldgraf** [comment]
A note that we had another person ask about Binder logo etc usage in the forum: https://discourse.jupyter.org/t/binder-logo-usage-in-plugin/3853

**Comment by betatim** [comment]
Nods. The reply that was given is a bit weird because mybinder/binder/the logos are not trademarks held by Numfocus so the trademark committee can't really tell them much :-/

**Comment by meeseeksmachine** [comment]
This issue has been mentioned on **Jupyter Community Forum**. There might be relevant details there:

https://discourse.jupyter.org/t/binder-logo-usage-in-plugin/3853/6
